docs/academy/herocourse/module2.md

```diff
@@ -6,6 +6,7 @@ Refer to the documentation references, given at the end of the each lesson, for
 
 ## Lesson 1: The Manifest File
 
+<br/>
```

Comment:
It provides some space above the video. Otherwise video is glued to a heading above and looks cluttered. 
I can easily revert this change if you think opposite @jamesbayly